Software/main/main.cpp

```diff
@@ -30,22 +30,21 @@ int main() {
 
     // TODO: flight loop
     for (int count  = 0; count < 100; count++) {
-        for (int i = 0; i < sensors.size(); i++) {
-            // Polls data from sensor #i
+        for (int i = 0; i < sensors.size(); ++i) {
```

Comment:
why?

Reply:
No reason--spaces vs tabs. Changed to i++ for loop increment consistency.